mava/utils/logger_tools.py

```diff
@@ -13,34 +13,28 @@
 # limitations under the License.
 
 import logging
-import os
-from collections import defaultdict
-from typing import Dict, List, Tuple
+from typing import Dict
 
 import neptune
 from colorama import Fore, Style
-from neptune.integrations.sacred import NeptuneObserver
-from sacred import Experiment, observers, utils
-from sacred.run import Run
+from neptune.utils import stringify_unsupported
 
 
 class Logger:
-    """Logger class for logging to tensorboard, and sacred.
+    """Logger class for logging to tensorboard, and neptune.
 
     Note:
         For the original implementation, please refer to the following link:
         (https://github.com/uoe-agents/epymarl/blob/main/src/utils/logging.py)
     """
 
-    def __init__(self, console_logger: logging.Logger) -> None:
+    def __init__(self, cfg: Dict) -> None:
         """Initialise the logger."""
-        self.console_logger = console_logger
+        self.cfg = cfg
+        self.console_logger = get_python_logger()
 
         self.use_tb = False
-        self.use_sacred = False
-
-        # defaultdict is used to overcome the problem of missing keys when logging to sacred.
-        self.stats: Dict[str, List[Tuple[int, float]]] = defaultdict(lambda: [])
+        self.use_neptune = False
```

Comment:
Why not grab `use_tb` and `use_neptune` from the config and set it here, then we don't even need to store the config